### PR TITLE
display the minimum rank only

### PR DESCRIPTION
**Pull request by poluxxs**
Only the minimal rank is displayed.

Big plus this speeds the calculation. However we have to display the rank 0 (sky) on the frontEnd now to see the topo.

### CODE REVIEW

File: geocruncher/ComputeIntersections.py [hunk 1]
```diff
@@ -25,7 +25,10 @@ def computeRank(a,b):
             evaluate_z = topography.evaluate_z
             ranks=model.rank
             return ranks([a,b,evaluate_z([a,b])])
-
+            
+        def findMin(a,b):
+            return np.minimum(a,b)
```
Comment (phile314-fh) [inline geocruncher/ComputeIntersections.py]: Is this method really useful? Why not just call ``np.minimum`` directly where it is used?

File: geocruncher/ComputeIntersections.py [hunk 2]
```diff
@@ -106,7 +109,10 @@ def computeRank(x,z):
             y=slope*(x-xCoord[0])+yCoord[0]
             ranks=model.rank
             return ranks([x,y,z])
-
+            
+        def findMin(a,b):
```
Comment (phile314-fh) [inline geocruncher/ComputeIntersections.py]: dito

**Comment by poluxxs** [comment]
done

**Comment by phile314-fh** [comment]
There was also a question:
> Maybe it would make sense to handle the sky differently and pick the other unit in that case?

**Comment by poluxxs** [comment]
@phile314-fh  I asked arnauld: He want to see only the topo on top and the interface between units only in color, so the current way make sense for him

File: geocruncher/ComputeIntersections.py [hunk 3]
```diff
@@ -180,8 +180,8 @@ def computeRankMatrix(index):
         yBoundaryList = yBoundaryList[xBoundaryList!=-1]
         xBoundaryList= xBoundaryList[xBoundaryList!=-1]
 
-        xBoundaryList=((xBoundaryList-(xCoord[0]))*(imgSize[1]))/((xCoord[1])-(xCoord[0])) #coord conversions
-        yBoundaryList = imgSize[0]-((yBoundaryList-(zCoord[0]))*(imgSize[0])/((zCoord[1])-(zCoord[0])))  #coord conversions
+        xBoundaryList=((xBoundaryList-(xCoord[0]))*(imgSize[1]))/((xCoord[1])-(xCoord[0])) + offSet #coord conversions
```
[the remark this answers is not in the document]
Comment (poluxxs) [reply geocruncher/ComputeIntersections.py]: It is the restriction to still use 0-1 on the whole CS.

If the begining of the CS insinde the project bounds is at 0.2 we need a 0.2 offSet to still be between 0-1 and not 0-0.8

**Comment by poluxxs** [comment]
@phile314-fh is there any more question or is it okay?

**Pull request merged**